From: Helmut Grohne <helmut@subdivi.de>
Date: Fri, 15 Dec 2017 23:33:05 -0500
Subject: fix pkg-config cross-compilation

Do not force a wrong value of PKG_CONFIG onto PKG_CHECK_MODULES.
It knows how to find the right pkg-config.

Closes: #858092
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 8d4d77f..f33b174 100644
--- a/configure.ac
+++ b/configure.ac
@@ -16,7 +16,6 @@ AC_PROG_CPP
 AC_PROG_INSTALL
 AC_PROG_LN_S
 AC_PROG_MAKE_SET
-AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
 
 GETTEXT_PACKAGE=xdg-user-dirs-gtk
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [The gettext domain name])
